subsys/partition_manager/pm.yml.huk_nrf52840

```diff
@@ -0,0 +1,8 @@
+#include <autoconf.h>
+
+# Use the last partition in flash to store the hardware unique key(HUK)
+# The size of the partition matches the fprotect block size since it will
+# be locked by fprotect.
```

Comment:
This is not strictly locked by fprotect, you're using acl directly, but I would actually favor extending the api of fprotect to support read-protection, and using that.

Reply:
That's correct, depending on what we will decide to do with fprotect I will change that accordingly.